pallets/manta-sbt/src/lib.rs

```diff
@@ -146,6 +216,20 @@ pub mod pallet {
     #[pallet::storage]
     pub(super) type NextSbtId<T: Config> = StorageValue<_, StandardAssetId, OptionQuery>;
 
+    /// Account that can add evm accounts to allowlist
+    #[pallet::storage]
+    pub(super) type AllowlistAccount<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
+
+    /// Allowlist for Evm Accounts
+    #[pallet::storage]
+    pub(super) type EvmAddressAllowlist<T: Config> =
+        StorageMap<_, Blake2_128Concat, EvmAddressType, MintStatus, OptionQuery>;
+
+    /// Range of time at which evm mints for each `MintType` are possible.
+    #[pallet::storage]
+    pub(super) type MintTimeRange<T: Config> =
+        StorageMap<_, Blake2_128Concat, MintType, (T::Moment, Option<T::Moment>), OptionQuery>;
```

Comment:
Not sure parachain based timestamp is safe enough here.

Reply:
We don't need very precise timestamp onchain. one or two block time error is fine.